staging/src/k8s.io/mount-utils/resizefs_linux.go

```diff
@@ -16,22 +16,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package resizefs
+package mount
 
 import (
 	"fmt"
-
 	"k8s.io/klog/v2"
-	"k8s.io/mount-utils"
 )
 
 // ResizeFs Provides support for resizing file systems
 type ResizeFs struct {
-	mounter *mount.SafeFormatAndMount
+	mounter *SafeFormatAndMount
 }
 
 // NewResizeFs returns new instance of resizer
-func NewResizeFs(mounter *mount.SafeFormatAndMount) *ResizeFs {
+func NewResizeFs(mounter *SafeFormatAndMount) *ResizeFs {
```

Comment:
Would it be possible to have `utilexec.Interface` as the only parameter? Because that's the only thing that `ResizeFs` needs. And smaller requirements in general make better interfaces.

You will need to refactor `SafeFormatAndMount.GetDiskFormat` - create a separate function `getDiskFormat(exec utilexec.Interface, disk string)` with all the logic, and have it called both by `mounter.GetDiskFormat` and `ResizeFs.Resize()`.

Perhaps as a separate commit in this PR.

Reply:
Done.